web/themes/joinup/templates/form/details.html.twig

```diff
@@ -14,24 +14,26 @@
  * @see template_preprocess_details()
  */
 #}
-<div class="vertical-tabs__menu-item vertical-tabs__menu-item--mobile">
-  <a href="#{{ element['#id'] }}">
-    <span class="vertical-tabs__menu-item-title">{{ title }}</span>
-    <span class="vertical-tabs__menu-item-summary">{{ description }}</span>
-  </a>
-</div>
 <details{{ attributes }}>
+  {%
+    set summary_classes = [
+      required ? 'js-form-required',
+      required ? 'form-required',
+      'form__label',
+    ]
+  %}
   {%- if title -%}
-    <summary{{ summary_attributes }}>{{ title }}</summary>
```

Comment:
Was checking the [whitespace control](https://twig.symfony.com/doc/2.x/templates.html#whitespace-control) in the operator, can you also please add it in the title if the control is really necessary?